cmd/secret-wrapper/main.go

```diff
@@ -172,6 +172,18 @@ func execCmd(argv []string) error {
 	proc := exec.Command(argv[0], argv[1:]...)
 	proc.Stdout = os.Stdout
 	proc.Stderr = os.Stderr
+	// provide a writeable home so kubectl discovery can be cached
+	home, set := os.LookupEnv("HOME")
+	needHome := !set
+	if set {
+		if err := syscall.Access(home, syscall.O_RDWR); err != nil {
+			// $HOME is set but not writeable
+			needHome = true
+		}
+	}
+	if needHome {
+		proc.Env = append(os.Environ(), "HOME=/alabama")
```

Comment:
To save time for other potential paranoid reviewers:
```
// If Env contains duplicate environment keys, only the last
// value in the slice for each duplicate key is used.
```

Reply:
Fixed in the following PR